tests/dist/src/test/java/org/hawkular/apm/tests/dist/AnalyticsServiceITest.java

```diff
@@ -108,7 +109,7 @@ public void initTest() {
     public void testGetUnboundEndpoints() throws Exception {
         Trace trace1 = new Trace();
         trace1.setId("1");
-        trace1.setStartTime(System.currentTimeMillis() - 4000); // Within last hour
+        trace1.setStartTime(TimeUnit.MILLISECONDS.toMicros(System.currentTimeMillis()) - 4000); // Within last hour
```

Comment:
The 4000 would need to be converted to microseconds  - although not sure why it says "within last hour" - but currently it represented 4 seconds, so needs to be 4000000 to be the same. However if the test works, then possibly not an issue.

If a fix is applied, then needs to be applied multiple times in this file.